BankSDK/GiniBankSDK/Sources/GiniBankSDK/Resources/GiniColorScheme.swift

```diff
@@ -0,0 +1,256 @@
+//
+//  GiniColorSchemeV2.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+import GiniCaptureSDK
+
+public struct GiniColorScheme {
+    struct Bg {
+        var primary: GiniColor
+        var secondary: GiniColor
+    }
+
+    struct ActionSheets {
+        var buttonsBg: GiniColor
+        var cancelButtonBg: GiniColor
+    }
+
+    struct Alert {
+        var bg: GiniColor
+        var divider: GiniColor
+    }
+
+    struct NavigationBar {
+        var bg: GiniColor
+        var action: GiniColor
+        var navigation: GiniColor
+    }
+
+    struct Content {
+        var navigation: GiniColor
+        var text: GiniColor
+    }
+
+    struct BottomBar {
+        var bg: GiniColor
+        var border: GiniColor
+    }
+    
+    struct Card {
+        var bg: GiniColor
+    }
+
+    struct Placeholder {
+        var bg: GiniColor
+        var tint: GiniColor
+    }
+
+    struct Text {
+        var primary: GiniColor
+        var secondary: GiniColor
+        var tertiary: GiniColor
+        var accent: GiniColor
+        var success: GiniColor
+        var error: GiniColor
+    }
+
+    struct Icons {
+        var primary: GiniColor
+        var secondary: GiniColor
+    }
+
+    struct PageController {
+        var selected: GiniColor
+        var unselected: GiniColor
+    }
+
+    struct Error {
+        var bg: GiniColor
+        var icon: GiniColor
+    }
+
+    struct Message {
+        var bgSuccess: GiniColor
+        var contentSuccess: GiniColor
+        var bgWarning: GiniColor
+        var contentWarning: GiniColor
+        var bgError: GiniColor
+        var contentError: GiniColor
+    }
+
+    struct Badge {
+        var bg: GiniColor
+        var content: GiniColor
+    }
+
+    struct Button {
+        var bg: GiniColor
+        var bgLoading: GiniColor
+        var content: GiniColor
+    }
+
+    struct ButtonOutlined {
+        var bg: GiniColor
+        var content: GiniColor
+    }
+
+    struct TextField {
+        var bg: GiniColor
+        var border: GiniColor
+        var focusedText: GiniColor
+        var unfocusedText: GiniColor
+        var disabledText: GiniColor
+        var errorText: GiniColor
+        var labelFocused: GiniColor
+        var labelUnfocused: GiniColor
+        var labelDisabled: GiniColor
+        var labelError: GiniColor
+        var supportingFocused: GiniColor
+        var supportingUnfocused: GiniColor
+        var supportingDisabled: GiniColor
+        var supportingError: GiniColor
+        var trailingFocused: GiniColor
+        var cursorEnabled: GiniColor
+        var cursorError: GiniColor
+    }
+
+    struct Toggles {
+        var thumb: GiniColor
```

Comment:
[clarification] What is thumb? We should use better and more descriptive names.

Reply:
its a "control" of iOS UISwitch, it has an option to set `thumbTintColor`